### PR TITLE
Update subaccounts.md to include info about subaccount reporting users

### PR DESCRIPTION
**Pull request by haleysolomon**
Added info about subaccount reporting users.

### CODE REVIEW

File: support/user-guide/subaccounts.md [hunk 1]
```diff
@@ -49,7 +50,8 @@ In addition, your subaccount users will be able to:
 * Get raw message event data via the API
 * View and manage the suppression list via the API​
 * Create their own webhook via the API
-* Create, edit, view, and use templates
+* Create, edit, view, and use templates via the API
+* Access analytics and event data, including Signals, via the UI
```
Comment (jgzamora) [inline support/user-guide/subaccounts.md]: @haleysolomon shouldn't this be just "Access Signals analytics through the UI"?

File: support/user-guide/subaccounts.md [hunk 2]
```diff
@@ -102,7 +104,9 @@ After creating a subaccount, the master account has the option of assigning a de
 
 **Permissions**
 
-Only master account users can create subaccounts. If you create a app user, they will have access to both the master account and subaccounts, so it is critical that be taken into account when provisioning app accounts on your system.
+Master account Admin and Developer users can create subaccounts. Master account Templates and Reporting users can see information about all subaccounts. Reporting Users can optionally be assigned to a specific subaccount instead of the master account - more information on this below in "Subaccount UI Access". It is important to consider if a user should have access to the master and all subaccounts or if a user should be restricted to a specific subaccount when provisioning app accounts on your system.
```
Comment (jgzamora) [inline support/user-guide/subaccounts.md]: imo you can get rid of the last sentence. Alternatively: "provisioning app accounts on your system" -> "inviting users".

File: support/user-guide/subaccounts.md [hunk 3]
```diff
@@ -211,4 +217,18 @@ Sending domain and tracking domain functionality afforded to subaccount users ar
 
 **Note**: When making API requests for the above reports, the subaccount does not need to specify the subaccount ID in their payloads. Even if this is included, no matter the value, it will always be overwritten since the subaccount ID is tied to the API key associated with it. Due to this restriction, a subaccount will not be able to spoof a different subaccount ID and get access to any other subaccount's or master account's data or assets.
 
+### Subaccount UI Access
+*Limited access. Reach out to our support team or your Technical Account Manager to have this option enabled for your account.
+
+Users with the "Reporting" role can be assigned to a specific subaccount. These users only have access to information about that subaccount within the SparkPost UI. They can access their email analytics, including SparkPost Signals, as well as their events. They also have read-only access to templates and snippets associated with their specific subaccount or shared with all subaccounts.
```
Comment (jgzamora) [inline support/user-guide/subaccounts.md]: again would just call it all "Signals analytics" or "analytics and event data"

File: support/user-guide/subaccounts.md [hunk 4]
```diff
@@ -211,4 +217,18 @@ Sending domain and tracking domain functionality afforded to subaccount users ar
...
+
+Other details to keep in mind:
+* Subaccount Reporting users can see their subaccount name and ID in the app.
+* A Reporting user can only be assigned to a subaccount at account creation. If you want to assign a current user to a specific subaccount, you will need to delete and re-add the user.
+* A Reporting user cannot be switched to a different subaccount or be changed to a the master account. If you want to make these changes, you will need to delete and re-add the user.
+* A Reporting user can only be assigned to a single subaccount. Multi-subaccount access is not supported at this time.
```
Comment (jgzamora) [inline support/user-guide/subaccounts.md]: I think this line was only appropriate in early beta docs, not public docs.

File: support/user-guide/subaccounts.md (hunk 1 above)
Comment (haleysolomon) [reply support/user-guide/subaccounts.md]: I was a little hesitant to assume everyone new what that was. I changed the wording and linked to a doc with more info, that make me feel better :-D

File: support/user-guide/subaccounts.md (hunk 2 above)
Comment (haleysolomon) [reply support/user-guide/subaccounts.md]: I didn't love that last sentence either, i waffled on taking it out or leaving it in. Agreed on taking it out.

File: support/user-guide/subaccounts.md (hunk 3 above)
Comment (haleysolomon) [reply support/user-guide/subaccounts.md]: updated

File: support/user-guide/subaccounts.md (hunk 4 above)
Comment (haleysolomon) [reply support/user-guide/subaccounts.md]: remove the last sentence

**Pull request merged**